Clinical-Genomics/scout
# On diagnoses page, show number of OMIM terms available and datetime of last update

Fix #3241 

To achieve that, I've replaced the OMIM terms `_id` of type string (example OMIM:67655) with a pymongo ObjectID

From this:

<img width="500" alt="image" src="https://user-images.githubusercontent.com/28093618/184314926-450941c3-bc69-4683-9ace-c8c741067200.png">


To this:

<img width="525" alt="image" src="https://user-images.githubusercontent.com/28093618/184315042-c4422c0e-fa9b-4a5c-b5e5-eed6d6c8d138.png">


<details>
<summary>Testing on cg-vm1 server (Clinical Genomics Stockholm)</summary>

**Prepare for testing**
1. Make sure the PR is pushed and available on [Docker Hub](https://hub.docker.com/repository/docker/clinicalgenomics/scout-server-stage)
1. Fist book your testing time using the paxa software installed on `hasta`:
 - Log in into hasta: `ssh <USER.NAME>@hasta.scilifelab.se`
 - Activate the staging environment with the command `us` (Use Stage)
 - Run the command `paxa` and follow the instructions. Make sure to specify scout-stage as the resource you request and the server cg-vm1 as server.
1. Log out from the hasta server.
1. `ssh <USER.NAME>@cg-vm1.scilifelab.se`
1. `sudo -iu hiseq.clinical`
1. `ssh localhost`
1. (optional) Find out which scout branch is currently deployed on cg-vm1: `podman ps`
1. Stop the service with current deployed branch: `systemctl --user stop scout.target`
1. Start the scout service with the branch to test: `systemctl --user start scout@<this_branch>`
1. Make sure the branch is deployed: `systemctl --user status scout.target`
1. After testing is done, log out from `cg-vm1` and log in again in the `hasta` server, repeat the `hasta` and `paxa` procedure, which will release the allocated resource (scout-stage) to be used for testing by other users.
</details>


**How to test locally**:
1. Locally, run `scout setup demo`
2. Go to diagnoses page

**How to test on hasta and cg-vm1**:
1. Install the branch on cg-vm1 and surf the pages which might be affected by changes in OMIM terms document structure
- assign and removing OMIM terms from cases
- Go to OMIM diagnoses page (should crash and only show number of OMIM terms, no last update date)
- Create and edit phenomodels containing OMIM terms
2. Install the branch on hasta stage and update the OMIM terms
3. Repeat 1

**Expected outcome**:
The functionality should be working
Take a screenshot and attach or copy/paste the output.

**Review:**
